XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs

```diff
@@ -471,44 +492,39 @@ public virtual void OnInputChanged(InputEventData<Vector2> eventData)
 
             if (isNudgePossible)
             {
-                var pointers = eventData.InputSource.Pointers;
+                pointers = eventData.InputSource.Pointers;
 
-                for (int i = 0; i < pointers.Length; i++)
-                {
-                    var newExtent = pointers[i].PointerExtent;
-                    var currentRaycastDistance = pointers[i].Result.Details.RayDistance;
 
-                    // Reset the cursor extent to the nearest value in case we're hitting something close
-                    // and the user wants to adjust. That way it doesn't take forever to see the change.
-                    if (currentRaycastDistance < newExtent)
-                    {
-                        newExtent = currentRaycastDistance;
-                    }
+                // We assume all pointers currently have the same length
+                var newExtent = pointers[0].PointerExtent;
```

Comment:
since we already know the pointer that's started the manipulation, should we just modify that one and leave the rest alone?

What is the benefit of adjusting all the pointers from this input source to do the nudging?